### ISSUE TITLE
Datetime deserialization requires Python 3.7+; emit useful error on earlier Python versions

### ISSUE THREAD

**Issue by Kartstig** [chanzuckerberg/sqlalchemy-aurora-data-api#14]
When loading an object with a DateTime from the DB I get the following error:
```
[2020-08-06 21:27:11,628] ERROR in app: Error in user confirmation: type object 'datetime.datetime' has no attribute 'fromisoformat'
type object 'datetime.datetime' has no attribute 'fromisoformat': AttributeError
Traceback (most recent call last):
  File "/var/task/app.py", line 48, in handle_post_authentication
    raise e
  File "/var/task/app.py", line 25, in handle_post_authentication
    .filter_by(sub=event["request"]["userAttributes"]["sub"])
  File "/var/task/sqlalchemy/orm/query.py", line 3316, in one_or_none
    ret = list(self)
  File "/var/task/sqlalchemy/orm/loading.py", line 101, in instances
    util.raise_from_cause(err)
  File "/var/task/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/var/task/sqlalchemy/util/compat.py", line 153, in reraise
    raise value
  File "/var/task/sqlalchemy/orm/loading.py", line 81, in instances
    rows = [proc(row) for row in fetch]
  File "/var/task/sqlalchemy/orm/loading.py", line 81, in <listcomp>
    rows = [proc(row) for row in fetch]
  File "/var/task/sqlalchemy/orm/loading.py", line 574, in _instance
    populators,
  File "/var/task/sqlalchemy/orm/loading.py", line 695, in _populate_full
    dict_[key] = getter(row)
  File "/var/task/sqlalchemy_aurora_data_api/__init__.py", line 75, in process
    return self.py_type.fromisoformat(value) if isinstance(value, str) else value
AttributeError: type object 'datetime.datetime' has no attribute 'fromisoformat'
```

Here is my User model:
![image](https://user-images.githubusercontent.com/2031422/89584558-44935400-d80a-11ea-85a2-60d0d77e4674.png)

**Comment by kislyuk** [comment]
The `fromisoformat` method is new in Python 3.7. If you wish to retrieve datetimes from your table, you must use Python 3.7+. Re-titled issue to reflect this.

**Comment by kislyuk** [comment]
Fixed in b5c22da7b9d5b0e9bb15a50121b31a73534ada47, released in v0.2.1.
